metadatastore/examples/sample_data/common.py

```diff
@@ -69,28 +69,3 @@ def noisy(val, sigma=0.01):
         return val + sigma * np.random.randn()
     else:
         return val + sigma * np.random.randn(len(val)).reshape(val.shape)
-
-
-get_time = ttime.time
-
-
-def example(func):
```

Comment:
(Now there is some code duplication in the individual examples, but there are only two examples and soon we want to replace the sample data generation with actual bluesky anyway.)